lib/custom_markdown_renderer.rb

```diff
@@ -87,6 +84,21 @@ def make_youtube_embed(youtube_match)
     )
   end
 
+  def make_loom_embed(loom_match)
+    video_id = loom_match[1]
+    %(
+      <iframe
+        src="https://www.loom.com/embed/#{video_id}"
+        width="640"
+        height="360"
```

Comment:
Can we do a responsive design?

Reply:
@pranavrajs We can make the video responsive based on layout, but we maintain a fixed width and keep the size of YouTube/Vimeo videos the same.